### PR TITLE
Add binance loom ethereum

### PR DESCRIPTION
**Pull request by andreipope**

### CODE REVIEW

File: docs/en/binance-loom-ethereum.md [hunk 1]
```diff
@@ -0,0 +1,512 @@
...
+  EventBus.$emit('updateStatus', { currentStatus: 'Withdrawing from Rinkeby Gateway.' })
+  await this._withdrawCoinsFromRinkebyGateway(data)
+}
+```
+
+Again, this function is pretty straightforward. First, it transfers the tokens to the transfer gateway contract on the Loom side:
+
+```js
+async _transferCoinsToExtdevGateway (amount) {
+    const amountInt = amount * 100000000
```
Comment (enlight) [inline docs/en/binance-loom-ethereum.md]: Use `BN` for any kind of token arithmetic, don't want to teach people bad habits.

File: docs/en/binance-loom-ethereum.md [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+      new BN(amountInt, 10),
+      tokenAddress,
+      ownerMainnetAddr
+    )
+    console.log('before receiveSignedWithdrawalEvent')
+    await receiveSignedWithdrawalEvent
+  }
+```
+
+One thing to note. As seen above, calling `await gatewayContract.withdrawERC20Async` creates a pending withdrawal. The pending withdrawal is picked up by the Gateway Oracle, which signs the withdrawal, and notifies the  Gateway. The Gateway emits an event to let the user know they can withdraw their token from the Ethereum Gateway to their Ethereum account by providing the signed withdrawal record.
```
Comment (enlight) [inline docs/en/binance-loom-ethereum.md]: The pending withdrawal is signed by the Gateway validators, and event is emitted by the Gateway to notify users a withdrawal has been signed. At that point the user can fetch the signed withdrawal receipt from the Gateway and submit it to the Ethereum Gateway in order to withdraw the token to their Ethereum account.

File: docs/en/binance-loom-ethereum.md [hunk 3]
```diff
@@ -0,0 +1,512 @@
...
+
+- lastly, let's withdraw the tokens to our Rinkeby account by calling the `withdrawCoinsFromRinkebyGateway` function and passing it the `data` structure containing the signature:
+
+
+```js
+async _withdrawCoinsFromRinkebyGateway (data) {
+  const rinkebyContractAddress = rinkebyBEP2Token.networks[this.rinkebyNetworkConfig['networkId']].address
+  const userRinkebyAddress = this.accountMapping.ethereum.local.toString()
+  const tx = await this.rinkeby2ExtdevGatewayContract.methods
+    .withdrawERC20(data.amount.toString(), data.signature, rinkebyContractAddress)
```
Comment (enlight) [inline docs/en/binance-loom-ethereum.md]: This is the API for the old gateway, won't work with the current extdev gateway.

File: docs/en/binance-loom-ethereum.md [hunk 4]
```diff
@@ -0,0 +1,512 @@
...
+      .send({ from: userRinkebyAddress, gas: '489362' })
+  } catch (error) {
+    console.log('Failed to transfer coin to the Ethereum Gateway')
+    throw error
+  }
+  EventBus.$emit('updateStatus', { currentStatus: 'Tokens deposited!' })
+}
+```
+
+First, it approves the transfer gateway to take the tokens and then calls the `depositERC20` method on the Rinkeby transfer gateway. The method expects two parameters: the amount to transfer and the address of the contract.
```
Comment (enlight) [inline docs/en/binance-loom-ethereum.md]: `address of the token contract`

File: docs/en/binance-loom-ethereum.md [hunk 5]
```diff
@@ -0,0 +1,512 @@
...
+We assume you've covered the first two parts of our Binance series. If so, let's make sure your local copy of the [Loom Examples repo](https://github.com/loomnetwork/loom-examples) is updated. You must `cd` into the `loom-examples` directory and run:
+
+```bash
+git pull
+```
+
+Next, let's make sure your dependencies are up-to-date:
+
+```text
+npm install
```
[the remark this answers is not in the document]
Comment (andreipope) [reply docs/en/binance-loom-ethereum.md]: @enlight I can make a different PR for this.

[the remark this answers is not in the document]
Comment (enlight) [reply docs/en/binance-loom-ethereum.md]: Probably not that important right now

**Pull request merged**